Formula/webfs.rb

```diff
@@ -19,14 +19,20 @@ class Webfs < Formula
 
   depends_on "openssl@1.1"
 
+  on_linux do
+    depends_on "httpd" => :build
```

Comment:
Can this be `uses_from_macos`? Otherwise I think we should always use httpd since the macOS apache has been deprecated for years.

Reply:
I did not realize macOS Apache was deprecated.  `httpd` is not available in the `uses_from_macos` allowlists, so I'm going to change this to depend on `httpd` on macOS as well.

Reply:
This is fixed now.